Classification/doc/Classification/Classification.txt

```diff
@@ -478,23 +508,29 @@ The following example:
 
 \subsection Classification_example_ethz_random_forest ETHZ Random Forest
 
-The following example shows how to use the classifier [ETHZ_random_forest_classifier](@ref CGAL::Classification::ETHZ_random_forest_classifier) using an input training set.
+The following example shows how to use the classifier [ETHZ::Random_forest_classifier](@ref CGAL::Classification::ETHZ::Random_forest_classifier) using an input training set.
 
 \cgalExample{Classification/example_ethz_random_forest.cpp}
 
 \subsection Classification_example_opencv_random_forest OpenCV Random Forest
 
-The following example shows how to use the classifier [OpenCV_random_forest_classifier](@ref CGAL::Classification::OpenCV_random_forest_classifier) using an input training set.
+The following example shows how to use the classifier [OpenCV::Random_forest_classifier](@ref CGAL::Classification::OpenCV::Random_forest_classifier) using an input training set.
 
 \cgalExample{Classification/example_opencv_random_forest.cpp}
 
+\subsection Classification_example_tensorflow_neural_network TensorFlow Neural Network
+
+The following example shows how to use the classifier [TensorFlow::Neural_network_classifier](@ref CGAL::Classification::TensorFlow::Neural_network_classifier) using an input training set.
```

Comment:
backticks missing

Reply:
What do you mean? Backticks are used to create autolinks, here I'm creating an explicit link (because I don't want the full name with namespaces to show), so backticks are not needed unless I'm mistaken.

Reply:
backtick = typewriting [`toto`](mylink) AFAIK

Reply:
(you need to click on edit, I can verbatim the toto link :) )

Reply:
Yes you can:
```
[`toto`](mylink)
```

Reply:
It doesn't work, if I do
```
[`toto`](mylink)
```
then it actually shows the backticks around the link in the produced doc. If I do
```
`[toto](mylink)`
```
then it shows the unaltered markdown code…